### ISSUE TITLE
Book Reviews

### ISSUE THREAD

**Issue by AlexWoollends** [gcallah/history_of_science#19]
Assigned myself to do HTML edits on the following reviews: ['Heavens and Heresies' about Kepler](https://www.unz.org/Pub/AmConservative-2004oct11-00033), and ['The Scientistic Revolution The Invention of Science: A New History of the Scientific Revolution'](https://www.questia.com/magazine/1P3-4052472951/the-scientistic-revolution) to myself.

**Comment by gcallah** [comment]
I don't think you have a way to get the second review. TAC never published that one either, and Unz did not archive it. 

But that one I do have a disk. As an MS Word file. Thus I think I had better convert it. 

Regards,
Gene Callahan

**Comment by AlexWoollends** [comment]
OK Gene.  Let me know.  I'll start searching the site for other posts while you do this.

**Comment by gcallah** [comment]
You can do Kepler I believe.

Kepler.html
